src/source/worker_source.js

```diff
@@ -11,7 +11,7 @@ export type TileParameters = {
 };
 
 export type WorkerTileParameters = TileParameters & {
-    coord: { z: number, x: number, y: number, w: number },
+    tileID: { overscaledZ: number, wrap: number, canonical: { z: number, x: number, y: number } },
```

Comment:
I see `OverscaledTileID` is [registered](https://github.com/mapbox/mapbox-gl-js/blob/c0ded76f6c4fd4f87fe4e38ee36d6bf1ea6f6520/src/util/web_worker_transfer.js#L52), so can this just be `tileID: OverscaledTileID`?